server/views/applications/pages/funding-information/funding-source.njk

```diff
@@ -1,14 +1,34 @@
 {% extends "../layout.njk" %}
 {% block questions %}
-  <h1 class="govuk-heading-l">{{ page.questions.fundingSource }}</h1>
-
   {{
     formPageRadios(
       {
+       fieldset: {
```

Comment:
So page.items was magically doing all of this for us before? And now the question is simpler we can remove that layer of indirection to simplify it by having it straight in the view here?